leader-election-impl/src/main/java/com/palantir/leader/PaxosLeaderElectionService.java

```diff
@@ -106,8 +109,33 @@ public PaxosLeaderElectionService(PaxosProposer proposer,
                                       long updatePollingWaitInMs,
                                       long randomWaitBeforeProposingLeadership,
                                       long leaderPingResponseWaitMs,
-                                      PaxosLeaderElectionEventRecorder eventRecorder,
                                       Supplier<Boolean> onlyLogOnQuorumFailure) {
+        this(proposer,
+                knowledge,
+                otherPotentialLeadersToHosts,
+                acceptors,
+                learners,
+                executorServiceFactory,
+                updatePollingWaitInMs,
+                randomWaitBeforeProposingLeadership,
+                DEFAULT_NO_QUORUM_MAX_DELAY_MS,
+                leaderPingResponseWaitMs,
+                PaxosLeaderElectionEventRecorder.NO_OP,
+                onlyLogOnQuorumFailure);
+    }
+
+    PaxosLeaderElectionService(PaxosProposer proposer,
```

Comment:
why do we have this *second* constructor if we have a builder? :S

Reply:
Since we do need to pass in the value of `leaderPingResponseWaitMs`, it's needed if you don't want to break the API. An alternative (which I'd prefer) is that we _can_ remove the deprecated constructor and leave just this one; a quick check on large internal product did not reveal any uses.

Reply:
yeah, understandable, just feel like we should be a bit more aggressive with breaks to make our codebase a bit cleaner one constructor at a time aha.